Bottom: 26e16c7f7e116dcdb44c8b66b80c52574b309282
Top:    3f335d7649816dcaf9a90ecc90d32983cc2385c4
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-28 03:23:13 +0100

drc: HACK HACK HACK


---

diff --git a/src/Makefile.am b/src/Makefile.am
index 4943755..353160a 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -285,6 +285,7 @@ libgtk_a_CPPFLAGS = -I./hid/gtk
 LIBGTK_SRCS = \
 	dolists.h \
 	hid/hidint.h \
+	hid/gtk/bga-wizard.c \
 	hid/gtk/gtkhid-main.c \
 	hid/gtk/gtkhid.h \
 	hid/gtk/gui.h \
